Lib/shutil.py

```diff
@@ -73,8 +73,13 @@ class RegistryError(Exception):
     and unpacking registries fails"""
 
 
-def copyfileobj(fsrc, fdst, length=16*1024):
-    """copy data from file-like object fsrc to file-like object fdst"""
+def copyfileobj(fsrc, fdst, length=None):
+    """Copy data from file-like object `fsrc` to file-like object `fdst`.
+
+    An in-memory buffer size can be set with `length`; the default is 16 KB.
```

Comment:
"An in-memory buffer size in bytes can be set withlength; the default is 16 KB."